test/UnitTest/TestingControllers/MessageboxInstancesControllerTests.cs

```diff
@@ -137,6 +137,29 @@ public async void GetMessageBoxInstance_RequestsInstanceUserIsNotAuthorized_Forb
             Assert.Equal(HttpStatusCode.Forbidden, responseMessage.StatusCode);
         }
 
+        [Fact]
+        public async void GetMessageBoxInstance_ArchivedInstanceCanBeCopied_UserHaveRights_InstanceIsSuccessfullyMappedAndReturned()
+        {
+            // Arrange
+            string instanceId = "1337/07274f48-8313-4e2d-9788-bbdacef5a54e";
```

Comment:
given that the instance is marked as hardDeleted in the test data it should not be returned with the get request. bug in the mock I assume. But let's update the testdata to reflect it being archived